graphistry/text_utils.py

```diff
@@ -285,5 +291,9 @@ def load_search_instance(self, savepath):
         from joblib import load  # type: ignore   # need to make this onnx or similar
 
         cls = load(savepath)
-        cls.build_index()
+        try:
+            cls.build_index()
+        except Exception as e:
```

Comment:
is there a way for us to know whether to build the index or not vs just doing it? current form is unclear how to maintain and handle